src/Mvc/test/WebSites/TagHelpersWebSite/Views/Home/MyHtml.cshtml

```diff
@@ -0,0 +1 @@
+﻿@(new TagHelpersWebSite.MyHtmlContent(Html))
```

Comment:
Could this just use `@Html.Partial("_Test")`? I'm not sure if the extra indirection is necessary.

Reply:
The extra indirection seems to be necessary. Tested it out and we don't get the Synchronous complain otherwise.